### DISCUSSION TITLE
Intl.Numberformat, currencyDisplay: "narrowSymbol" not supported

### DISCUSSION THREAD

**Discussion by damiangene** [vercel/next.js#16498] [Help]
I'm using Next.JS 9.4.4

When trying to use:
```TS
new Intl.NumberFormat('en-GB', { style: 'currency', currency: currency, useGrouping: true, currencyDisplay: 'narrowSymbol'});
```
I receive the error:
```
RangeError: Value narrowSymbol out of range for Intl.NumberFormat options property currencyDisplay
    at new NumberFormat
```

narrowSymbol is a supported property as you can see [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat) and I have used it successfully on another project that wasn't using Next

**Comment by youminkim** [comment]
Same issue. It happens in old iOS device such as iPhone 8
